qemu/tests/cfg/numa.cfg

```diff
@@ -3,6 +3,15 @@
     kill_vm_on_error = yes
     login_timeout = 240
     numa_hardware_cmd = "numactl --hardware"
+    mem_ratio = 0.6
+    mem_map_tool = "mem_mapping.tar.gz"
+    stress_cmds_mem_mapping = "./mem_mapping"
+    cmd_cp_mmap_tool = "/bin/cp -rf %s %s && cd %s && tar zxvf ${mem_map_tool}"
+    make_cmds_mem_mapping = "gcc mem_mapping.c -o ${stress_cmds_mem_mapping}"
+    cmd_mmap = "cd %s/mem_mapping && ${make_cmds_mem_mapping} && numactl -m %s ${stress_cmds_mem_mapping} -a -p -l %dK &"
+    cmd_mmap_cleanup = "rm -rf %s/mem_mapping*"
+    cmd_mmap_stop = "pkill -9 mem_mapping"
+    cmd_migrate_pages = "migratepages %s %s %s"
```

Comment:
Some configuration are for host which be used in `numa_stress` only.  Please be careful with this, thanks!

Reply:
Updated, thanks !